### PR TITLE
[PHEE-597] Automate create and activate voucher step in integration test.

### PR DESCRIPTION
**Pull request by logoutdhaval**
## Description

[PHEE-597 Automate create and activate voucher step in integration test.](https://fynarfin.atlassian.net/browse/PHEE-597?atlOrigin=eyJpIjoiOGEyZjhhNDcxMTc3NDkwNjkyMGE1ZTkxMTZmOWEwOWIiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!
- [x] Followed the PR title naming convention mentioned above.

- [x] Acknowledge that we will not merge PRs that are not passing the checks ("green") - it is your (author's) responsibility to get a proposed PR to pass all the checks, not primarily the project's maintainers.

- [x] The PR title should include a JIRA ticket

- [ ] Design-related bullet points or design document links related to this PR added in the description above.

- [ ] Updated corresponding Postman Collection or API documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [x] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

- [x] Followed coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

FYI our guidelines for code reviews same as https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: loadTest.csv [hunk 1]
```diff
@@ -0,0 +1,514 @@
+"voucherNumber","serialNumber"
+"axrhYm227/yOLmsGkIzFIFyYnzYgRooptni0LmByc8s/QvihWaGcjUlRef+KMomq54jjqqUnuO56hg3vhZRoMwav7KK+hFjcWXh7MxRA7kMiOniqSxFrpCVxPjSs2JeSsd7N/oy6p7OAGsM92PiGaHKlgLcvx/uKirWlPgE9CBRwQRVLnr4CIX2OBgrKPrbbimnQo/MbNlKUGQGr8BHro/eXHPsf8gYml9xCKvkZBA7wmo9SHAha9/uRAP5gzUalyi+bCldqhdsSR68pSBlazZ16+AAL1wnNfBQbeyRm7qEuWtqlmJGvDSSiCeeIsGDhhlaiZE9brG0LzjcJDZ88Qw==","17085922872146"
```
Comment (fynmanoj) [inline loadTest.csv:2]: generated files need not be version controlled, ignore this file

File: vouchers_data.csv
Comment (fynmanoj) [inline vouchers_data.csv:1]: Keep this file in a separate directory and add a readme file to document how to use this

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: loadTest.csv (hunk 1 above)
Comment (logoutdhaval) [reply loadTest.csv]: Added in .gitignore

File: vouchers_data.csv
Comment (logoutdhaval) [reply vouchers_data.csv:1]: added readme.md

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**